config/foreman.migrations/160614145332_foreman_expire_hosts.rb

```diff
@@ -0,0 +1,2 @@
+answers['foreman::plugin::expire_hosts'] ||= false
```

Comment:
This file should be named 20160614145332... so it orders correctly with the existing files.